Source/WebCore/rendering/svg/RenderSVGText.cpp

```diff
@@ -99,7 +99,7 @@ const RenderSVGText* RenderSVGText::locateRenderSVGTextAncestor(const RenderObje
 static inline void collectLayoutAttributes(RenderObject* text, Vector<SVGTextLayoutAttributes*>& attributes)
 {
     for (RenderObject* descendant = text; descendant; descendant = descendant->nextInPreOrder(text)) {
-        if (auto* svgInline = dynamicDowncast<RenderSVGInlineText>(*descendant))
+        if (CheckedPtr svgInline = dynamicDowncast<RenderSVGInlineText>(*descendant))
```

Comment:
Not strictly needed since `layoutAttributes()` is a trivial inline function. If this is hot code, we could consider not making this change.

Reply:
Ah, sorry, did not check layoutAttributes properly.